accounts-kethereum/src/main/java/pm/gnosis/svalinn/accounts/utils/TransactionExtensions.kt

```diff
@@ -57,10 +56,10 @@ fun Transaction.Legacy.rlp(signature: ECDSASignature? = null): ByteArray {
     items.add((value?.value ?: BigInteger.ZERO).toRLP())
     items.add((data?.hexStringToByteArray() ?: ByteArray(0)).toRLP())
     if (signature != null) {
-        items.add(adjustV(signature.v).toRLP())
+        items.add(if(chainId > BigInteger.ZERO) adjustV(signature.v).toRLP() else signature.v.toRLP())
```

Comment:
The `chainId > BigInteger.ZERO` is already in https://github.com/5afe/svalinn-kotlin/pull/60/files#diff-5e36b314c4c1d48db4d3d472ff4a6f17b90d6a7c91a32375f3e6c8fa9b6b0de4L76 no need to do it here again, right?